app/src/main/java/com/orgzly/android/data/DataRepository.kt

```diff
@@ -1163,11 +1162,15 @@ class DataRepository @Inject constructor(
         })
     }
 
-    fun updateNoteContent(bookId: Long, noteId: Long, content: String?) {
+    fun updateNoteContent(noteId: Long, content: String?) {
```

Comment:
Nice. But I really wonder why your solution (getting bookId from the note) was not used before...